docs/configuration/alertmanager-integration/_pull_integration.rst

```diff
@@ -3,25 +3,27 @@ Configure Metric Querying
 
 Metrics querying lets Robusta pull metrics and create silences.
 
-To configure it, add the following to ``generated_values.yaml`` and :ref:`update Robusta <Simple Upgrade>`.
+To configure it, add the ``prometheus_url`` and ``alertmanager_url`` to your Helm values and :ref:`update Robusta <Simple Upgrade>`.
```

Comment:
I would add a note here about auto detection. i.e.:
```
If Robusta fails to auto-detect the Prometheus and Alertmanager urls, and you see related connection errors in the logs,  configure the ``prometheus_url`` and ``alertmanager_url`` to your Helm values and :ref:`update Robusta <Simple Upgrade>`
```

Reply:
@arikalon1 I think it's best to avoid it here because this section is included in multiple pages, not just for robusta in-cluster. So it is also for external prometheuses etc when auto-detect is not relevant.

We could fix by changing the docs and not reusing snippets, but for now I suggest we just merge this as is.

Is that OK?

Reply:
auto-detect is mostly relevant for 3rd party prometheus and alert manager

Reply:
we try to auto detect all known prometheus distributions

Reply:
ok, updated